app/views/docs/authentication-magic.phtml

```diff
@@ -0,0 +1,196 @@
+<p>
+   Magic URL is a password-less way to authenticate users. 
+   When a user logs in, they will receive an email with a "magic" link that contains a secret used to log in the user.
+   The user can simply click the link to be logged in.
+</p>
+
+<h2><a href="#init" id="init">Send Email</a></h2>
+
+<p>
+    Initialize the log in process with the <a href="/docs/client/account#accountCreateMagicURLSession">Create Magic URL Session</a> route.
+    If the email has never been used, a <b>new user ID is generated</b>, then the user will receive an email. 
+    If the email is already attached to an account, the <b>user ID is ignored</b> and the user will receive a link in their email.
+</p>
+
+<ul class="phases clear" data-ui-phases>
+    <li>
+        <h3>Web</h3>
+        <div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+            <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Client, Account, ID } from "appwrite";
+
+const client = new Client()
+    .setEndpoint('https://cloud.appwrite.io/v1') // Your API Endpoint
+    .setProject('[PROJECT_ID]');               // Your project ID
+
+const account = new Account(client);
+
+
+const promise = account.createMagicURLSession('[USER_ID]', 'email@example.com');
```

Comment:
Can we provide `uniqe()` in all examples? I think that's common usage compared to custom IDs

Reply:
Yes